src/agents/parking_agent.py

```diff
@@ -25,119 +27,138 @@ def __init__(self, jid, password, x, y, parking_spots, parking_price=40):
             parking_spots for _ in range(24)
         ]  # a list with number of free/available spots at a given hour
         self._parking_price = parking_price
+        self._users_reservations = {}  # {user_jid: {reservation_id: [time_start, time_stop]}}
 
     def _prepare_check_parking_spots_template(self):
         template = Template()
-        template.to = f"{self.jid}@{DEFAULT_HOST}"
-        template.set_metadata("performative", "inform")
-        template.set_metadata("inform", "parking-availability")
+        template.to = f"{self.jid}"
+        template.set_metadata("performative", "query-ref")
+        template.set_metadata("action", "check-parking")
         return template
 
     def _prepare_make_reservation_template(self):
         template = Template()
-        template.to = f"{self.jid}@{DEFAULT_HOST}"
-        template.set_metadata("performative", "inform")
-        template.set_metadata("inform", "make-reservation")
+        template.to = f"{self.jid}"
+        template.set_metadata("performative", "request")
+        template.set_metadata("action", "make-reservation")
         return template
 
     def _prepare_modify_reservation_template(self):
         template = Template()
-        template.to = f"{self.jid}@{DEFAULT_HOST}"
-        template.set_metadata("performative", "inform")
```

Comment:
How about adding a table to README.md with details about the FIPS protocol? I can add it to this PR or another

Reply:
I'll do it in another PR